magicctapipe/io/io.py

```diff
@@ -1075,6 +1075,14 @@ def load_dl2_data_file(input_file, quality_cuts, event_type, weight_type_dl2):
     # Get the mean DL2 parameters
     df_dl2_mean = get_dl2_mean(event_data, weight_type_dl2)
     df_dl2_mean.reset_index(inplace=True)
+    event_before_nan_filter = df_dl2_mean.shape[0]
+    df_dl2_mean.dropna(inplace=True)
+    event_after_nan_filter = df_dl2_mean.shape[0]
+
+    logger.info(
+        f"{event_before_nan_filter - event_after_nan_filter} "
+        f"stereo events removed because of NaN values."
```

Comment:
for the Crab files that I was using for the tests I did not get a single event rejected like this, but I guess it must have happened to you that's why you introduce this check

Reply:
indeed, it was another sample, but it gave problems because some events had NaN for the reconstructed alt/az and ra/dec, and then it gave errors when multiplying by a unit.